packages/flutter_tools/lib/src/base/clock.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:quiver/time.dart';
+
+import 'context.dart';
```

Comment:
> They're generally less test-friendly.

Github comments are probably not a good venue for this discussion, but I'd be curious to know what makes them less test-friendly than anything else. Something tells me we're making different assumptions :)